docs/docs-next/docs/guides/automation/asset-sensors.md

```diff
@@ -1,10 +1,80 @@
 ---
-title: Asset Sensors
-sidebar_position: 50
+title: Scheduling pipelines across projects and jobs with Asset Sensors
+sidebar_label: Scheduling pipelines across projects and jobs
+sidebar_position: 30
 ---
 
-### Basic Asset Sensor Example
+Asset sensors in Dagster provide a powerful mechanism for monitoring asset materializations and triggering downstream computations or notifications based on those events.
+
+There are many use cases for asset sensors, however the most common is to trigger a job when an asset is materialized in a different job or code location, or to provide custom logic to determine when to trigger a run following an asset materialization.
+
+### Cross-job and cross-code location dependencies
+
+Asset Sensors enable dependencies across different jobs and even different code locations. This flexibility allows for more modular and decoupled workflows.
+
+
+```mermaid
+graph LR;
+
+AssetToWatch --> AssetSensor;
+AssetSensor --> Job;
+Job --> Asset1;
+Job --> Asset2;
+
+subgraph CodeLocationA
+    AssetToWatch
+end
+
+subgraph CodeLocationB
+    AssetSensor
+    Job
+    Asset1
+    Asset2
+end
+
+```
+
+Here's a minimal example of an asset sensor that triggers a job when an asset is materialized. The `daily_sales_data` asset is in the same code location for this example, but the same pattern can be applied to assets in different code locations.
 
 <CodeExample filePath="guides/automation/simple-asset-sensor-example.py" language="python" title="Simple Asset Sensor Example" />
 
-This Asset Sensor will trigger a run of `my_job` whenever the `asset_to_watch` asset is materialized.
+### Custom evaluation logic
+
+The evaluation function of an asset sensor can be customized to include custom logic for determining when to trigger a run. This allows for fine-grained control over the conditions under which downstream jobs are executed.
```

Comment:
Nit - `... asset sensor can be customized to include custom logic...`

Can we change up the word choice here so we don't have two variations of `custom` in the same sentence?